### DISCUSSION TITLE
when i add a tiff without srs,the tiff has become blurred

### DISCUSSION THREAD

**Discussion by csuxing** [pelicanmapping/osgearth#2027] [Support Q&A]
this is my code:
osgEarth::Drivers::FileSystemCacheOptions filecapoptions;
	filecapoptions.rootPath() = "D:\\OEcache";
	osgEarth::Map::Options configue;
	configue.cache() = filecapoptions;
	configue.cachePolicy() = osgEarth::CachePolicy::USAGE_READ_WRITE;

	osg::ref_ptr<osg::Group> root = new osg::Group;
	osg::ref_ptr<Map> map = new Map(configue, nullptr);

	osgEarth::ProfileOptions profileOpts;
	profileOpts.srsString() = "PROJCS["Beijing 1954 / 3-degree Gauss-Kruger CM 111E",GEOGCS["Beijing 1954",DATUM["Beijing_1954",SPHEROID["Krassowsky 1940",6378245,298.3,AUTHORITY["EPSG","7024"]],TOWGS84[15.8,-154.4,-82.3,0,0,0,0],AUTHORITY["EPSG","6214"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4214"]],PROJECTION["Transverse_Mercator"],PARAMETER["latitude_of_origin",0],PARAMETER["central_meridian",111],PARAMETER["scale_factor",1],PARAMETER["false_easting",500000],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AUTHORITY["EPSG","2434"]]";//
	profileOpts.bounds() = osgEarth::Bounds(ex.minx, ex.miny, ex.maxx, ex.maxy);//the tiff's bounds
	map->setProfile(Profile::create(profileOpts));

	osg::ref_ptr<osgEarth::MapNode> mapNode = new MapNode(map.get());

	{
		GDALImageLayer* localImageMap = new GDALImageLayer();
		localImageMap->setURL(sdipath);
		map->addLayer(localImageMap);
	}

**Comment by gwaldron** [comment]
It is hard to answer this question without knowing what you see and what you expect to see.

**Reply by csuxing** [reply]
Thanks for replying to my post.
This is I can see:
![from_oe](https://user-images.githubusercontent.com/52850824/160838860-7ab656e1-2513-41f4-b578-e445fe252728.jpg)
This is I want to see:
![from_ori](https://user-images.githubusercontent.com/52850824/160838929-d7b384bb-a09d-4095-a9c1-7f786ae5c37f.jpg)
It looks like the resolution of the picture has been reduced.

**Reply by csuxing** [reply]
the left is I want to see，the right is I can see
![cw](https://user-images.githubusercontent.com/52850824/160840124-1a3ebe78-775c-40a1-aab1-715e552e888d.jpg)

**Reply by csuxing** [reply]
(1) I have an image file with plane projection information,which is 'PROJCS["Beijing 1954 / 3-degree Gauss-Kruger CM 111E",GEOGCS["Beijing 1954",DATUM["Beijing_1954",SPHEROID["Krassowsky 1940",6378245,298.3,AUTHORITY["EPSG","7024"]],TOWGS84[15.8,-154.4,-82.3,0,0,0,0],AUTHORITY["EPSG","6214"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4214"]],PROJECTION["Transverse_Mercator"],PARAMETER["latitude_of_origin",0],PARAMETER["central_meridian",111],PARAMETER["scale_factor",1],PARAMETER["false_easting",500000],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AUTHORITY["EPSG","2434"]]'. 
(2) I set its srs info and border extension info：
osgEarth::ProfileOptions profileOpts;
profileOpts.srsString() = "PROJCS["Beijing 1954 / 3-degree Gauss-Kruger CM 111E",GEOGCS["Beijing 1954",DATUM["Beijing_1954",SPHEROID["Krassowsky 1940",6378245,298.3,AUTHORITY["EPSG","7024"]],TOWGS84[15.8,-154.4,-82.3,0,0,0,0],AUTHORITY["EPSG","6214"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4214"]],PROJECTION["Transverse_Mercator"],PARAMETER["latitude_of_origin",0],PARAMETER["central_meridian",111],PARAMETER["scale_factor",1],PARAMETER["false_easting",500000],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AUTHORITY["EPSG","2434"]]";//
profileOpts.bounds() = osgEarth::Bounds(ex.minx, ex.miny, ex.maxx, ex.maxy);//the tiff's bounds, The bounds range I got by reading the header information of the tiff file 
map->setProfile(Profile::create(profileOpts));
(3) It can be displayed in osgearth, but the highest precision pagedLOD layer has a gap with the original image. 
the left is I want to see,the right is I can see.
![cw](https://user-images.githubusercontent.com/52850824/161175697-7e36276f-fa5a-43a6-92fe-8561f2812db6.jpg)
(4)Why can't I get the original image resolution?
